selfdrive/car/tests/routes.py

```diff
@@ -116,6 +116,7 @@ class CarTestRoute(NamedTuple):
   CarTestRoute("6a42c1197b2a8179|2023-09-21--10-23-44", HYUNDAI.KIA_OPTIMA_H_G4_FL),
   CarTestRoute("c75a59efa0ecd502|2021-03-11--20-52-55", HYUNDAI.KIA_SELTOS),
   CarTestRoute("b3537035ffe6a7d6|2022-10-17--15-23-49", HYUNDAI.KIA_SPORTAGE_HYBRID_5TH_GEN),
+  CarTestRoute("428412ad82894ba3|2023-12-09--17-01-47", HYUNDAI.KIA_SPORTAGE_PHEV_5TH_GEN, segment=8),
```

Comment:
> can you put your device on WiFi and upload a short 10 minute route with some engagement? Then we can merge!

@sshane I see you've added `428412ad82894ba3|2023-12-09--17-01-47` segment 8 for testing. Would you still like a different route? I can upload another shorter route in the next 2 days if that is helpful

Reply:
That's fine, but can you get a route where you test the max lateral acceleration it will apply?

Reply:
> can you get a route where you test the torque/max lateral acceleration it will apply

Next time I'm near highway, I will try the highway interchange or exit suggestion to get the out of torque warning. 

In the meantime, in the `428412ad82894ba3|2023-12-17--17-23-36--6` [route](https://connect.comma.ai/428412ad82894ba3/1702851815530/1702852454596) at `17:28:13 – 4`, the road took a sharp turn and the OpenPilot oversteered. Not sure if this was due to poor visibility at night with rain and oncoming headlights though.

Reply:
Can you pull and try again? It should not be oversteering as it was.

Reply:
> Can you pull and try again?

What's been holding me up is that even though the C3X is configured to use the `kia-sportage-phev-2023-port` branch, it doesn't find the latest commit even when I manually push check. It still says "up to date" but shows a stale commit 42a8a6df98c4c25c49a7ceb21676640a6ab246a6 as the current version.

Reply:
> it doesn't find the latest commit even when I manually push check

Ah I think I understand the issue. I think my comma is checking the [`commaai/openpilot:kia-sportage-phev-2023-port`](https://github.com/commaai/openpilot/tree/kia-sportage-phev-2023-port) branch rather than the [`sunnyhaibin/openpilot-1:kia-sportage-phev-2023-port`](https://github.com/sunnyhaibin/openpilot-1/tree/kia-sportage-phev-2023-port) branch.

I will install from my fork that I sync and report back.

Reply:
> Can you pull and try again? It should not be oversteering as it was.

Okay, here are logs based on d44e81c2727c15b2cfc43211f2b4ccf4cbe2158e: `428412ad82894ba3|2023-12-30--13-15-10--8` and [route](https://connect.comma.ai/428412ad82894ba3/1703960109006/1703960618401). I didn't reach max lateral torque on this route, so let me know if we need to capture a highway exit.

Reply:
@sshane tagging for re-review with the the latest logs in this thread.

Reply:
My experience has been that it still oversteers slightly on long curves. Installed from your fork

Reply:
Okay, so @xtian and I find that there is a bit of oversteering on sharp turns with the 2023 Kia Sportage PHEV. 

@Wafflezzbutt [reports](https://github.com/commaai/openpilot/pull/30575#issuecomment-1949546120) the same on the 2024 Kia Sportage PHEV (with additional fingerprints added).

> I do feel like it overcorrects sometimes. It seems to "lean left" especially around turns, and then over correct back

So @sshane / @sunnyhaibin my questions are:

1. is the oversteering issue also present with Hybrid and Gas models of the Kia Sportage 2023+? If so I wonder if the best approach is to merge and then fix all models.
2. Independent of 1, how do we go about fixing the oversteeting?